web/src/types.ts

```diff
@@ -103,3 +103,9 @@ export type Snapshot = {
   SnapshotLink: string
   showSnapshotModal: boolean
 }
+
+export enum ShowFatalErrorModal {
+  Default,
+  Show,
```

Comment:
This value isn't used yet but I'd like, in the future, to provide a way to show the Fatal Error Modal even if it has been dismissed, if there is a fatal error.